python/GafferDispatchTest/StatsApplicationTest.py

```diff
@@ -61,7 +61,7 @@ def testContext( self ) :
 
 		o = subprocess.check_output(
 			[
-				"gaffer", "stats", script["fileName"].getValue(),
+				Gaffer.executablePath(), "stats", script["fileName"].getValue(),
```

Comment:
This needs the `str( Gaffer.executablePath() )` treatment as well.

Reply:
Ah, thanks for catching that - squashed in to https://github.com/GafferHQ/gaffer/pull/5016/commits/179634bde07d9bf2fd6df93f3e69a186223bed31.